frontend/src/routes/manage/Realm/Content/Edit/EditMode/Series.tsx

```diff
@@ -107,53 +102,26 @@ export const EditSeriesBlock: React.FC<EditSeriesBlockProps> = ({ block: blockRe
             onChange={data => seriesField.onChange(data?.id)}
             onBlur={seriesField.onBlur}
         />
-
-        <Heading>{t("series.settings.order")}</Heading>
-        <NiceRadio breakpoint={0}>
-            <NiceRadioOption
-                value="NEW_TO_OLD"
-                defaultChecked={order === "NEW_TO_OLD"}
-                {...form.register("order")}
-            >{t("series.settings.new-to-old")}</NiceRadioOption>
-            <NiceRadioOption
-                value="OLD_TO_NEW"
-                defaultChecked={order === "OLD_TO_NEW"}
-                {...form.register("order")}
-            >{t("series.settings.old-to-new")}</NiceRadioOption>
-        </NiceRadio>
-
-        <Heading>{t("manage.realm.content.series.layout.heading")}</Heading>
-        <LayoutChooser {...{ currentLayout, form }} />
+        <div css={{
+            display: "flex",
+            flexDirection: "row",
+            marginTop: 12,
+            marginRight: 48,
+            justifyContent: "space-between",
+            maxWidth: 400,
+            [screenWidthAtMost(BREAKPOINT_SMALL)]: {
+                flexDirection: "column",
+                gap: 12,
+            },
+        }}>
+            <div>
+                <Heading>{t("series.settings.order")}</Heading>
+                <DisplayOptionGroup type="radio" optionProps={orderProps} {...{ form }} />
+            </div>
+            <div>
+                <Heading>{t("manage.realm.content.series.layout.heading")}</Heading>
+                <DisplayOptionGroup type="checkbox" optionProps={layoutProps} {...{ form }} />
+            </div>
```

Comment:
Isn't there a grouping necessary for accessibility? The radio buttons and checkboxes are not part of a fieldset. I would expect some aria-label as described in `Approach 2` or a fieldset.
https://www.w3.org/WAI/tutorials/forms/grouping/

Reply:
Hm I think you are right, thank you for pointing it out.